tools/rna_tools/mlocarna/test-data/Makefile

```diff
@@ -0,0 +1,28 @@
+ALL_TESTS=archaea-default archaea-probabilistic haca.snoRNA-default
```

Comment:
is there a standard way to generate the test-output automatically? What if default parameter values change?
Currently, I generate the test output with this Makefile -- so, for me it is at least useful :)